addons/medical/scripts/Game/Damage/DamageEffects/CharacterDamageEffects/ACE_Medical_MorphineDamageEffect.c

```diff
@@ -0,0 +1,19 @@
+//------------------------------------------------------------------------------------------------
+class ACE_Medical_MorphineDamageEffect: SCR_DotDamageEffect
+{
```

Comment:
See: https://discord.com/channels/105462288051380224/976155351999201390/1244272008733134910
```suggestion
{

	//------------------------------------------------------------------------------------------------
	protected override void HandleConsequences(SCR_ExtendedDamageManagerComponent dmgManager, DamageEffectEvaluator evaluator)
	{
		super.HandleConsequences(dmgManager, evaluator);

		evaluator.HandleEffectConsequences(this, dmgManager);
	}

```

Reply:
This is still pointless, unless we provide an overloaded version of `HandleEffectConsequences`, which is not really needed, since we have nothing to handle.